### PR TITLE
98 fix python challenge update

### PR DESCRIPTION
**Pull request by yeografik**

### CODE REVIEW

File: app/python/views.py [hunk 1]
```diff
@@ -83,29 +85,44 @@ def create_new_challenge():
...
+
     req_challenge = PythonChallenge.query.filter_by(id=id).first()
-    
+
     if req_challenge is None:   #case id is not in database
         return make_response(jsonify({"challenge":"there is no challenge with that id"}),404)
 
     response = PythonChallenge.to_dict(req_challenge).copy()   #start creating response for the endpoint
-    
+    response.pop('best_score', None)
```
Comment (matiasg99) [inline app/python/views.py]: it does not seem a very good idea to take the best score

**Review by matiasg99 — CHANGES_REQUESTED** [review]
I recommend removing line  98

File: app/python/views.py (hunk 1 above)
Comment (yeografik) [reply app/python/views.py]: you are right, postman shows that best_score needs to be returned in response

**Review by matiasg99 — APPROVED** [review]
congrats

**Pull request merged**